students/pooria_k/Lesson06/good_perf.py

```diff
@@ -0,0 +1,46 @@
+"""
+poorly performing, poorly written module
+
+"""
+
+import datetime
+import csv
+
+def analyze(filename):
+    """Read CSV file line by line
+        Analyzie it, create year_count and count how manytimes
+        'ao' was found
+    """
+    year_count = {
+        "2013": 0,
+        "2014": 0,
+        "2015": 0,
+        "2016": 0,
+        "2017": 0,
+        "2018": 0
+    }
+    found = 0
+    start = datetime.datetime.now()
+    with open(filename) as csvfile:
+        reader = csv.reader(csvfile, delimiter=',', quotechar='"')
+        new_ones = []
```

Comment:
_new_ones_ variable is initialized but not used. You should be able to remove it.